src/main/java/com/exactpro/th2/FixHandler.java

```diff
@@ -565,12 +576,15 @@ public Map<String, String> onIncoming(@NotNull IChannel channel, @NotNull ByteBu
 
         if(receivedMsgSeqNum < serverMsgSeqNum.get() && !isDup) {
             if(settings.isLogoutOnIncorrectServerSequence()) {
-                context.send(CommonUtil.toEvent(String.format("Received server sequence %d but expected %d. Sending logout with text: MsgSeqNum is too low...", receivedMsgSeqNum, serverMsgSeqNum.get())));
-                sendLogout(String.format("MsgSeqNum too low, expecting %d but received %d", serverMsgSeqNum.get() + 1, receivedMsgSeqNum));
+                String text = debugAndFormat("Received server sequence %d but expected %d. Sending logout with text: MsgSeqNum is too low...", receivedMsgSeqNum, serverMsgSeqNum.get());
+                context.send(CommonUtil.toEvent(text));
+                text = debugAndFormat("MsgSeqNum too low, expecting %d but received %d", serverMsgSeqNum.get() + 1, receivedMsgSeqNum);
```

Comment:
I think we don't need to format message to be sent to gateway in Logout Text field